rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/kustomize/KustomizeBakeManifestRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.netflix.spinnaker.rosco.manifests.kustomize;
+
+import com.netflix.spinnaker.rosco.manifests.BakeManifestRequest;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class KustomizeBakeManifestRequest extends BakeManifestRequest {
+    String namespace;
```

Comment:
As far as I know, Kustomize doesn't have any CLI options that let us set this. I think we can omit this.

Reply:
removed